test/verify/check-docker-storage

```diff
@@ -84,19 +85,22 @@ class TestDockerStorage(MachineCase):
             b.wait_present("#containers-storage-details a")
 
     def testDevmapper(self, vg=""):
-        b = self.browser
+        m = self.machine
 
         # On the Atomics, we use two machines: one for running
         # cockpit-ws, and one whose docker storage pool is managed.
         # We can't do that both on a single machine since resetting
         # the pool would kill the cockpit-ws container.
 
         if self.machine.atomic_image:
-            m = self.new_machine(machine_key='additional')
-            m.start()
-            m.wait_boot()
+            login_machine = self.new_machine(machine_key='login',
+                                             image=get_build_image(self.machine.image))
```

Comment:
The main reason is speed, we need an image that has cockpit installed. We already have built rpms for that image so it makes it easy. The comment you have there is fine with me to add.